grow/ui/sass/tool_ga_inspect.sass

```diff
@@ -0,0 +1,19 @@
+.grow__tool__ga
+  &--active
+    [data-g-event]
+      background: repeating-linear-gradient(-45deg, #FBE9E7, #FFCCBC 5px, white 5px, white 10px)
+
+      &:hover::after
+        background: #E64A19
+        border-radius: .25em
+        bottom: 50px
+        color: #ffffff
+        content: '[Event] ' attr(data-g-event) ' [Action] ' attr(data-g-action) ' [Label] ' attr(data-g-label)
```

Comment:
Ok, I'll work on changing this PR to focus on using the settings to control tools. Then later when we work on GA in airkit we can update the `ga-inspect` tool to work with that as well.